internal/selectors/selectors.go

```diff
@@ -174,6 +174,18 @@ var exclusionByLabels = []ExclusionByLabelsSpec{
 	{Key: "cattle.io/creator", Value: "norman"},
 }
 
+// A annotation as a key, value pair, used to exclude resources matching this annotation
+type ExclusionByAnnotationsSpec struct {
+	Key   string
+	Value string
+}
+
+// ExclusionByAnnotationsSpec are known annotation key which are excluded from propagation. Right
+// now only used to exclude resources created by Openshift
+var exclusionByAnnotations = []ExclusionByAnnotationsSpec{
+	{Key: "openshift.io/description", Value: ""},
```

Comment:
```suggestion
	{Key: "openshift.io/description"},
```

Reply:
+1

Reply:
Ok ,
I thought about it and you are probably right. Better to give only the name, than to have ambiguity when giving an empty annotation verse of any annotation
 I will make changes